topcoffea/cfg/2018.cfg

```diff
@@ -7,82 +7,82 @@ verbose: 1
-#ttWH : TTWH_madgraph
-#TTTW : TTTW_madgraph
-#TTTJ : TTTJ, TTTJ_madgraph_ext1
-#ttGG : TTGG_0Jets_madspin
-#TTTW : TTTW_TuneCP5_madgraph_ext1
-
-tHq : THQ_4f_Hincl_madgraph
-ttHnobb : ttHToNonbb_M125_TuneCP5 #ttHJetToNonbb_M125_TuneCP5_amcatnloFXFX_madspin
-tZq : tZq_ll_4f_ckm_NLO_TuneCP5_madgraph_ext1
-
-#THW_5f_Hincl_madgraph
-#TTGG_0Jets_TuneCP5_amcatnlo_madspin
-#TTTJ_TuneCP5_madgraph_ext1
-#WWTo2L2Nu_DoubleScattering
-
-### Data
-# EGamma
-EGamma_2018 : EGamma_Run2018A_Nano25Oct2019_v1, EGamma_Run2018B_Nano25Oct2019_v1, EGamma_Run2018C_Nano25Oct2019_v1, EGamma_Run2018D_Nano25Oct2019_v1 : 10
-
-#MuonEG
-MuonEG_2018 : MuonEG_Run2018A_Nano25Oct2019_v1, MuonEG_Run2018B_Nano25Oct2019_v1, MuonEG_Run2018C_Nano25Oct2019_v1, MuonEG_Run2018D_Nano25Oct2019_ver2_v1 : 10
-
-#DoubleMuon
-DoubleMuon_2018 : DoubleMuon_Run2018A_Nano25Oct2019_v1, DoubleMuon_Run2018B_Nano25Oct2019_v1, DoubleMuon_Run2018C_Nano25Oct2019_v1, DoubleMuon_Run2018D_Nano25Oct2019_ver2_v1 : 10
-
-# Single Muon
-SingleMuon_2018 : SingleMuon_Run2018A_Nano25Oct2019_v1, SingleMuon_Run2018B_Nano25Oct2019_v1, SingleMuon_Run2018C_Nano25Oct2019_v1, SingleMuon_Run2018D_Nano25Oct2019_v1 : 10
-
+#TTZToLL_M_1to10 : TTZToLL_M_1to10_TuneCP5_amcatnlo
+#TTZToLLNuNu_M_10_a: TTZToLLNuNu_M_10_TuneCP5_amcatnlo_ext1
+
+#### Triboson
+#WWW : WWW_4F_TuneCP5_amcatnlo_ext1
+#WZG : WZG_TuneCP5_amcatnlo
+#WWZ : WWZ_TuneCP5_amcatnlo_ext1
+#WZZ : WZZ_TuneCP5_amcatnlo_ext1
+#ZZZ : ZZZ_TuneCP5_amcatnlo_ext1
+##ZZG : ZZGJetsTo4L2Nu_4f_TuneCP5_amcatnloFXFX
+#
+## tt raros
+#tttt : TTTT_TuneCP5_amcatnlo_ext1
+#ttWW : TTWW_TuneCP5_madgraph_ext1, TTWW_TuneCP5_madgraph_ext2
+#ttWZ : TTWZ_TuneCP5_madgraph_ext1
+#ttZH : TTZH_TuneCP5_madgraph_ext1
+#ttZZ : TTZZ_madgraph_ext1
+#ttHH : TTHH_TuneCP5_madgraph_ext1
+#ttZZ : TTZZ_TuneCP5_madgraph_ext1
+##ttWH : TTWH_madgraph
+##TTTW : TTTW_madgraph
+##TTTJ : TTTJ, TTTJ_madgraph_ext1
+##ttGG : TTGG_0Jets_madspin
+##TTTW : TTTW_TuneCP5_madgraph_ext1
+#
+#tHq : THQ_4f_Hincl_madgraph
+#ttHnobb : ttHToNonbb_M125_TuneCP5 #ttHJetToNonbb_M125_TuneCP5_amcatnloFXFX_madspin
+#tZq : tZq_ll_4f_ckm_NLO_TuneCP5_madgraph_ext1
+#
+##THW_5f_Hincl_madgraph
+##TTGG_0Jets_TuneCP5_amcatnlo_madspin
+##TTTJ_TuneCP5_madgraph_ext1
+##WWTo2L2Nu_DoubleScattering
+#
+#### Data
+## EGamma
+#EGamma_2018 : EGamma_Run2018A_Nano25Oct2019_v1, EGamma_Run2018B_Nano25Oct2019_v1, EGamma_Run2018C_Nano25Oct2019_v1, EGamma_Run2018D_Nano25Oct2019_v1 : 10
+#
+##MuonEG
+#MuonEG_2018 : MuonEG_Run2018A_Nano25Oct2019_v1, MuonEG_Run2018B_Nano25Oct2019_v1, MuonEG_Run2018C_Nano25Oct2019_v1, MuonEG_Run2018D_Nano25Oct2019_ver2_v1 : 10
+#
+##DoubleMuon
+#DoubleMuon_2018 : DoubleMuon_Run2018A_Nano25Oct2019_v1, DoubleMuon_Run2018B_Nano25Oct2019_v1, DoubleMuon_Run2018C_Nano25Oct2019_v1, DoubleMuon_Run2018D_Nano25Oct2019_ver2_v1 : 10
+#
+## Single Muon
+#SingleMuon_2018 : SingleMuon_Run2018A_Nano25Oct2019_v1, SingleMuon_Run2018B_Nano25Oct2019_v1, SingleMuon_Run2018C_Nano25Oct2019_v1, SingleMuon_Run2018D_Nano25Oct2019_v1 : 10
+#
```

Comment:
Is updating to the latest samples, or just the ones that were used to test this branch?

Reply:
hmm right, 2018.cfg is pointing to local files so shouldn't probably be in the commit. I'll remove it.